tests/foreman/ui/test_ldap_authentication.py

```diff
@@ -583,7 +584,8 @@ def test_positive_delete_external_roles(
                 ldapsession.architecture.search('')
             ldapsession.location.create({'name': location_name})
             if is_open('BZ:1851905'):
-                ldapsession.browser.execute_script("window.history.go(-1)")
+                ldapsession.browser.selenium.back()
```

Comment:
We can find a better approach to working around BZ 1851905.

I would suggest navigating to the page you expect to be on with `navigate_to` and assert against that fresh view.

It doesn't appear that there is a need to specifically test and replicate a workaround that's within the BZ and expected to be used.

The test is failing in PRT - with nothing returned by `search(location_name)` on the next line, so I don't believe this is resolving the failure.

Reply:
Yeah, I'm looking for the fix and trying multiple options. Thanks for shifting draft. Once the PRT give okay this will be good

Reply:
@mshriver `navigate_to` itself will not work because we are on the page which is blank and has no way it will get navigate it's own because there not the menu on the left side. I think the possible solution would be navigate using url like to bas url and then start again.